cfspopcon/formulas/power_crossing_separatrix/threshold_power.py

```diff
@@ -89,7 +92,29 @@ def calc_LI_transition_threshold_power(plasma_current: float, average_electron_d
     Returns:
         :term:`P_LI_thresh` [MW]
     """
-    return float(2.11 * plasma_current**0.94 * ((average_electron_density / 10.0) ** 0.65)) * scale
+
+
+    def _calc_AUG_LI_threshold(average_electron_density, magnetic_field_on_axis, scale):
+        return float(2 * 0.07 * (average_electron_density / 10) * (magnetic_field_on_axis / 2.4) ** 0.39) * scale
+    
+    def _calc_HubbardNF17_LI_threshold(average_electron_density, magnetic_field_on_axis, scale):
+       return float(0.162 * (average_electron_density / 10) * (magnetic_field_on_axis ** 0.262)) * scale
```

Comment:
@aarooshgr as far as I can tell from the slides, these formulas are missing the surface area (`S`) parameter

Reply:
fixed!